user_endpoints.md

```diff
@@ -0,0 +1,73 @@
+# User Endpoints
+The following endpoints are only available through OAuth2 Client Credentials Flow with an application with the `provision_users` scope.
+
+_Note: This API is scoped to your organization’s white label._
+
+**User Attributes:**
+
+```json
+{
+  “username”: “user12345”,
+  “email”: “joe.smith@example.com”,
+  “auth_token”: “abc123”,
+  “status”: “active”
+}
```

Comment:
Need to figure out how to make the syntax highlighting happy here

Reply:
You can use `javascript` as the code type but it's still going to complain. Not sure what it's looking for.

Reply:
Fixed.  Stupid smart quotes.